rest-api/src/main/resources/definitions/study_info.yml

```diff
@@ -0,0 +1,41 @@
+description: |
+    A study (an outside organization providing participants for a study).
```

Comment:
Might be worth changing the wording to differentiate this from a full view of the study. Could adjust to "An abridged summary of a study (...)."

Reply:
Changed the wording